Spectra v2.6.1 (#3168)
brainstormforce/ultimate-addons-for-gutenberg: readme.txt

== Changelog ==

= 2.6.1 - FRIDAY, 19th MAY 2003 =
* Fix: Admin Dashboard - Added missing i18n support for some strings.
* Fix: Admin Dashboard - 'Activate all' and 'Deactivate all' previously had no impact on the status of Animations extension.

> * Fix: Animations Extension - Resolved Error loading issue for dynamic blocks caused by animation attributes.

Reviewer: ```suggestion
* Fix: Animations Extension - Resolved an error loading dynamic blocks that was caused by animation attributes.
```

Reviewer 2: I believe it was meant to be this, the Error loading in single quotes, because that's what mentioned in the error message.

`* Fix: Animations Extension - Resolved 'Error loading' issue for dynamic blocks caused by animation attributes.`

Reviewer 3: Done here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3169
